src/sentry/constants.py

```diff
@@ -227,6 +227,7 @@ def get_all_languages():
     "sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyCondition",
     "sentry.rules.conditions.event_attribute.EventAttributeCondition",
     "sentry.rules.conditions.level.LevelCondition",
+    "sentry.rules.filters.newer_older.NewerOlderFilter",
```

Comment:
So including this here will cause it to be returned from the response correct? I guess this is fine since we're not using the list of filters yet

Reply:
yup it will get sent in the `ProjectRulesConfigurationEndpoint` but it is not used by the frontend so nothing should change.